examples/test_suite_virtual/test_case_run.py

```diff
@@ -19,42 +19,36 @@
 from pykiso.auxiliaries import aux_virtual, aux_udp  # isort:skip
 
 
-@pykiso.define_test_parameters(suite_id=1, case_id=6, aux_list=[aux_udp], run_timeout=2)
+@pykiso.define_test_parameters(suite_id=1, case_id=6, aux_list=[aux_udp])
```

Comment:
I think the top should not be changed. I would more expect a warning here instead of an error.
You could configure it so that the timeouts throw a warning they are used with the basicTest

Reply:
Now it a timeout is pass as parameter of a BasicTest it throw a warning